app/models/database/ProfileDB.scala

```diff
@@ -0,0 +1,48 @@
+package models.database
+
+import models.Profile
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Reads, _}
+
+/**
+  * Definition of the database profile model
+  * @param id is used as primary key and the identifier on database level
+  * @param confirmed
+  * @param email
+  * @param userId foreign key to define the corresponding user
+  */
+case class ProfileDB(
+                  id: Long,
+                  confirmed: Boolean,
+                  email: String,
+                  userId: Long
+                )
+
+
+object ProfileDB{
+
+  def mapperTo(
```

Comment:
Same same same

Reply:
mapperTo functions are needed from the slick table def's